### ISSUE TITLE
Creator appearing on subscriptions list when I'm not subscribed. 

### ISSUE THREAD

**Issue by g8m3** [futo-org/grayjay-android#217]
Grayjay is showing videos for a rumble channel called Crain & company, even though I'm not subscribed to them. I tried subscribing and unsubscribing, and reinstalling and recovering from a manual backup. It's really annoying, because it's a sports channel (I despise spectator sports) and they put out a thousand videos a day, more than every other creator I subscribe to combined, so my subscription list is about 3/4 videos I couldn't care less about. Please advise.

**Comment by beemandev** [comment]
Temp workaround might be to actually sub to them in GrayJay then in settings for that creator, deselect all fetch settings

**Comment by g8m3** [comment]
Subscribed, deselected everything from fetch settings and saved, refreshed
subs list, no joy. Videos still appear on subs list.

**Comment by JustCryen** [comment]
I have exactly opposite issue. I'm subscribed to a creator with videos and streams selected, but I can't see his video he uploaded 2 hours ago even though, I got notified about it on other communication channel and I've already watched it.
It just doesn't show on the subscription feed or the home tab.

These issues are so polar opposite that I was unsure if I should create a new one or comment here instead.
